api/src/main/java/org/openmrs/module/patientqueueing/model/PatientQueue.java

```diff
@@ -67,6 +68,12 @@
 	@JoinColumn(name = "queue_room")
 	private Location queueRoom;
 	
+	@Column(name = "date_picked")
+	private Date datePicked;
+	
+	@Column(name = "date_completed")
```

Comment:
Can you also explain what this field is used for?

Reply:
Date  time Completed is used to have a clear status when the queue of a patient was completed. 
When a patient and provider session has ended and clinician is sending the patient out they have got to complete the session. and Date time Completed is very key. 

It can be used to compute serving time.

Reply:
@dkayiwa done